data/applications.yml

```diff
@@ -106,12 +106,6 @@
   team: Worldwide Publishing
   product_manager: Rob Rankin
 
-- github_repo_name: govuk_content_api
```

Comment:
This needs to stay, but you can add a `retired: true` flag, add a description what it used to be and remove team/PM:

https://github.com/alphagov/govuk-developer-docs/blob/master/data/applications.yml#L213-L223

That way the page will not 404 and we keep a bit of a record about apps, should that be needed.